QuizApp/Constants/Constants.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Constants.swift
+//  QuizApp
+//
+//  Created by Ana on 11/12/24.
+//
+
+import UIKit
+
+struct Constants {
+    
+    struct Colors {
+        static let yellowPrimary = UIColor.yellowPrimary
+        static let backgroundWhite = UIColor.white
+        static let textWhite = UIColor.white
+    }
```

Comment:
are you using colours from assets. its better to create UIColor extension for hex colours and use them from figma. colours in assets makes sense if you are using dark mode. If you will have questions we can call